oozie-to-airflow/templates/email.tpl

```diff
@@ -0,0 +1,13 @@
+{{ task_id }} = email_operator.EmailOperator(
+    task_id='{{ task_id }}',
+    trigger_rule='{{ trigger_rule }}',
+    params=PARAMS,
+    to={{ to }},
+    subject={{ subject }},
+    html_content={{ body }},
+    cc={{ cc|default(None, true) }},
+    bcc={{ bcc|default(None, true) }},
+    mime_subtype='{{ content_type|default('mixed', true) }}',
+    files=read_hdfs_files({{ attachment|default([], true) }})
```

Comment:
I'm new to this repo, so excuse my silly questions.

Does this mean that the output DAG will be dependent on this oozie-to-airflow lib? That sounds like it makes intuitive sense IMO.

Reply:
In a sense, but only the `o2a_libs/` folder. The way it's currently set up is that the o2a_libs/ directory must be present in the user's DAG folder. This will allow airflow to pick it up and use it on all the converted DAGs.